pkg/util/mount/nsenter_mount.go

```diff
@@ -229,3 +230,13 @@ func (n *NsenterMounter) absHostPath(command string) string {
 	}
 	return path
 }
+
+func (n *NsenterMounter) MakeRShared(path string) error {
+	nsenterCmd := nsenterPath
+	nsenterArgs := []string{
+		"--mount=/rootfs/proc/1/ns/mnt",
```

Comment:
It's hardcoded in whole nsenter mounter... I'll make a constant for it, it's used on too many places, but any serious refactoring should happen in a different PR.